docs/user-guide/mkdocs.yml

```diff
@@ -1,22 +1,41 @@
 docs_dir: docs-source
 site_name: AWS Ambit Scenario Designer for Unreal Engine 4 User Guide
 nav:
-    - Home: index.md
-    - Ambit Weather: weather.md
-    - Working with AWS in Ambit: aws-setup.md
-    - Individual Scenario Generation: individual-scenario-generation.md
-    - Bulk Scenario Generation: bulk-senario-generation.md
-    - Export Map Options: map-export.md
-    - Ambit Spawner Tools:
+    - Welcome:
+        - Overview: index.md
+    - Getting Started: 
+        - Setup: setup.md
+    - Static Obstacles:
         - Overview: spawner-parent.md
         - Spawn on Surface: spawn-on-surface.md
         - Spawn in Volume: spawn-in-volume.md
         - Spawn with Houdini Digital Assets: spawn-with-houdini.md
         - Spawn on Path: spawn-on-path.md
         - Ambit Spawner Examples: spawner-examples.md
-    - Ambit Path Generation Tools:
+    - Dynamic Obstacles: 
         - Overview: spawn-vehicle-path-parent.md
         - Spawn Vehicle Path: spawn-vehicle-path.md
+    - City Generation:
+        - Overview: city-generation.md
+        - Obtaining OpenStreetMap Files: obtaining-osm.md
+        - Building Generator: building-generator.md
+        - Road Generator: road-generator.md
+    - Weather & Time of Day FX:
+        - Overview: weather.md
+    - Simulation Scenarios:
+        - Manual Scenario Generation: individual-scenario-generation.md
+        - Automated Scenario Generation: bulk-senario-generation.md
+        - Working with AWS in Ambit: aws-setup.md
```

Comment:
Consider moving to top of this section.

Reply:
This whole section needs some reworking. I'll be doing that in the next PR. Out of scope for this PR.

Reply:
This is fine. Although, keep in mind the `Working with AWS in Ambit` section should be utilized by not just the scenarios as the maps use it, too. We can get to this later, though.